File: src/java/me/indexyz/strap/define/CommandContext.java
package me.indexyz.strap.define;

import me.indexyz.strap.object.Update;
import me.indexyz.strap.utils.BotNetwork;

import java.util.List;

public class CommandContext implements AbstractContext {
    public BotNetwork network;
    public List<String> args = null;
    public Update update;
}

File: src/kotlin/me/indexyz/strap/define/context/CommandContext.kt
package me.indexyz.strap.define.context

import me.indexyz.strap.`object`.Update
import me.indexyz.strap.utils.Configuration
import me.indexyz.strap.utils.Network
import java.util.*

class CommandContext(
        network: Network,
        update: Update,
        configuration: Configuration,
        var args: Optional<List<String>>
): MessageContext(
    network, update, configuration
)